homeassistant/components/light/strings.json

```diff
@@ -36,6 +36,30 @@
     "field_xy_color_name": "XY-color",
     "section_advanced_fields_name": "Advanced options"
   },
+  "conditions": {
+    "is_off": {
+      "description": "Test if a light is off.",
+      "description_configured": "Test if a light is off",
+      "fields": {
+        "behavior": {
+          "description": "How the state should match on the targeted lights.",
+          "name": "Behavior"
```

Comment:
Same comment as for the triggers; the user will see a select named "Behavior" with the description "How the state should match on the targeted lights." and options "All", "Any".
I think this is really difficult to understand.

Maybe it would be clearer to write "Lights which must be turned off for the condition to pass"